Google.PowerShell/Compute/GceCmdlet.cs

```diff
@@ -19,6 +20,9 @@ public abstract class GceCmdlet : GCloudCmdlet
         // The Servcie for the Google Compute API
         public ComputeService Service { get; }
 
+        // The value of the progress bar when being used as a spinner.
+        private int _spinnerValue;
```

Comment:
When being used as a spinner? Can it be used as anything else?

Why not, "Value of the current progress bar for cmdlets which have an infinite spinner to fake progress."